.eslintrc.json

```diff
@@ -17,6 +17,7 @@
   },
   "rules": {
     "no-console": ["error", { "allow": ["error"] }],
+    "react/prop-types": "off", // Typescript does this better
```

Comment:
The rule's documentation talks about understanding TypeScript. What's complaining?

Reply:
Mmmmm actually it looks like nothing, now. In a previous iteration of this, it was upset about my accessing a property on an object named `props` in `OutputSpec` (if I remember correctly, which I may not be, it wasn't even the actual component props, just another object created in the render function named `props`). Anyway, I'll undo this, since it's not needed anymore. If it comes up again we can evaluate